microservices/object_detection/detector.py

```diff
@@ -1,42 +1,67 @@
 from config import PATH_TO_MODEL, THRESHOLD
 import detector_base
+from logger import logger
 import detect
 from annotator import Annotator
 import bottle
-from bottle import route, run,  get, post, request 
+from bottle import route, run,  get, post, request, response
 import io
 from PIL import Image
 from PIL import ImageDraw
+import time
 
+error_plus_log = logger(name='error_plus_monitor',filename='api_error_plus.log')
+
+performance_log = logger(name='api_performance_monitor',
+                                  filename='api_perfomance.log', 
+                                  fmt='%(message)s')
+
+
+# need improvement
+performance_log.info('load_tflite_t,load_image_t,preprocess_image_t,invoke_tensors_t,detect_t,annotate_t')
+  
 @route('/')
 def greetings():
-    return "Hi, I am the AI4Coral RESTful API\n"
-
+  return "Hi, I am the AI4Coral RESTful API\n"
 
 @post('/detect/<filename>')
 def do_detections(filename):
-    try:
-        try:    
-            interpreter = detector_base.make_interpreter(PATH_TO_MODEL)
-            interpreter.allocate_tensors()
-
-            image = Image.open(request.body)
-            scale = detect.set_input(interpreter, image.size,
+  try:
+    start = time.perf_counter()
+    interpreter = detector_base.make_interpreter(PATH_TO_MODEL)
+    interpreter.allocate_tensors()
+    load_tflite_t = 1000 * (time.perf_counter() - start)
```

Comment:
This multiplication *1000 everywhere reduces the readability of the code. The print function that you have down below can take care of conversion or formating.

Reply:
solved